dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/config/DcesDrcIntegrationAuthenticationConfig.java

```diff
@@ -23,12 +23,11 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                                 .requestMatchers("/swagger-ui/**").permitAll()
                                 .requestMatchers("/api-docs/**").permitAll()
                                 .requestMatchers("/actuator/**").permitAll()
-                                .requestMatchers(HttpMethod.GET, API_REQUEST_PATH).permitAll()
+                                //.requestMatchers(HttpMethod.GET, API_REQUEST_PATH).permitAll()
```

Comment:
Is this commented out line being kept to easily swap between for devs? Or can we can it?

Reply:
If it's going to be removed, can it not be done now?